apex_launchtest/apex_launchtest/util/__init__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from sys import executable as __executable
+from launch.actions import ExecuteProcess as __ExecuteProcess
+
+
+def EmptyNode():
```

Comment:
@pbaughman qq, why the `Node` denomination?

Reply:
@hidmic When I had ROS stuff mixed in, this launched one of the empty example nodes because it was already getting installed in the share/examples directory [and it was a one-liner](https://github.com/ApexAI/apex_rostest/blob/master/apex_rostest/apex_rostest/util/__init__.py).  When I pulled out the ROS stuff, I just grepped for `launch_ros` and `rclpy` and didn't think to rename the method. I have no problem changing it to EmptyProc

Reply:
KeepAliveProc might be a better name, since that's a better description of what it's used for